readme.markdown

```diff
@@ -374,11 +374,24 @@ $ node object.js test/x.js test/y.js
 
 # install
 
-With [npm](https://npmjs.org) do:
+With [npm](https://npmjs.org) proceed through the following steps:
 
+- run npm init and set up your package.json: this tells you what node modules others need when working on your repo
+```
+npm init
+```
+- install
 ```
 npm install tape --save-dev
 ```
+- go to [gitignore.io](https://www.gitignore.io/) and type __NODE__
```

Comment:
this definitely isn't necessary, as git knowledge is also a prerequisite for doing development in general - and `jspm_packages` only applies if you're using JSPM.